packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -451,13 +451,21 @@ export function createProjectionNode<I>({
                                 delta,
                                 hasOnlyRelativeTargetChanged
                             )
-                            this.startAnimation({
+
+                            const animationOptions = {
                                 ...getValueTransition(
                                     layoutTransition,
                                     "layout"
                                 ),
                                 onComplete: onLayoutAnimationComplete,
-                            })
+                            }
+
+                            if (visualElement.shouldReduceMotion) {
+                                animationOptions.delay = 0
```

Comment:
I don't understand why we set the delay to 0

Reply:
Ah because usually `delay` is set (or derived from `staggerChildren`/`delayChildren`) to work in tandem with `duration`, which is now essentially `0`